src/api/models/User.ts

```diff
@@ -91,6 +92,9 @@ User.init(
                 isEmail: true
             },
             unique: true
+        },
+        bio: {
+            type: DataTypes.STRING
```

Comment:
You should either put a `defaultValue: 'something'` or `allowNull: false` option